model/PerpendicularPaths.py

```diff
@@ -71,33 +71,29 @@ def robot_by_cell (self, cell):
                 return r;
 
     def space_touched_by_xy (self, cell):
-        color = 15
+        color = 15 if os.name == 'nt' else ''
         for i, s in enumerate(self.space_touched):
             if s[1] == cell:
                 color = s[2]
         return color
 
+    def color_update (self, color, bgcolor=15 if os.name == 'nt' else 0):
+        if os.name == 'nt':
+            if self.std_output_hdl is None:
```

Comment:
Goal 1 of 5: move Red to cell [10,4]
         _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
        |Traceback (most recent call last):
  File "perpendicularpaths.py", line 361, in <module>
    game.play_game()
  File "perpendicularpaths.py", line 314, in play_game
    self.display_update()
  File "perpendicularpaths.py", line 117, in display_update
    self.color_update (color, back_color)
  File "perpendicularpaths.py", line 82, in color_update
    if self.std_output_hdl is None:
AttributeError: 'PerpendicularPaths' object has no attribute 'std_output_hdl'


aka need to add it as an attribute at the top of the class as std_output_hdl = None